### PR TITLE
docs(NonIdealState): Add Timed Out state

### PR DESCRIPTION
**Pull request by FlyersPh9**
- Adding **Timed out** state as requested by Diego Sosa.

[Related PR in iTwinUI-icons repo.](https://github.com/iTwin/iTwinUI-icons/pull/20)

Screen Shot 2021-10-12 at 4 01 37 PM

### CODE REVIEW

**Comment by FlyersPh9** [comment]
Unsure why, but more of the test images are failing than expected. It should only be 2 fails (1 new test for light & dark).
Screen Shot 2021-10-12 at 4 03 03 PM

File: backstop/scenarios/non-ideal-state.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ module.exports = [
   scenario('502', { selectors: ['#nis-502 .iui-non-ideal-state'] }),
   scenario('503', { selectors: ['#nis-503 .iui-non-ideal-state'] }),
   scenario('error', { selectors: ['#nis-error .iui-non-ideal-state'] }),
+  scenario('timed-out', { selectors: ['#nis-timed-out .iui-non-ideal-state'] }),
```
[the remark this answers is not in the document]
Comment (FlyersPh9) [reply backstop/scenarios/non-ideal-state.js:11]: I tried reordering, no luck.

I started playing with the misMatchThreshold.  If I set them all to `misMatchThreshold: 99` it reduces to only 6 failures. 😅

**Comment by bentleyvk** [comment]
If this was requested by someone, maybe just approve new test images and then we could merge it.

**Comment by mayank99** [comment]
> If this was requested by someone, maybe just approve new test images and then we could merge it.

There is no change in the component. The new illustration is already in the `illustrations-react` package.

**Review by mayank99 — APPROVED** [review]

**Pull request merged**